draft-thomson-quic-tls.md

```diff
@@ -320,7 +335,10 @@ At each point, both keying material (see {{key-expansion}}) and the the AEAD
 function used by TLS is interchanged with the values that are currently in use
 for protecting outbound packets.  Once a change of keys has been made, packets
 with higher sequence numbers MUST use the new keying material until a newer set
-of keys (and AEAD) are used.
+of keys (and AEAD) are used. TODO: Actually this doesn't work with QUIC
+retransmissions in which retransmissions consume the next sequence number. In
+particular, consider the retransmission of a handshake packet. It must be sent
+with the correct keys so that the peer can decrypt it.
```

Comment:
There is an exception for the handshake packets.  But for other key updates, this is the simplest rule.